### PR TITLE
Add basic etcd insufficient members runbook

### PR DESCRIPTION
**Pull request by lilic**
This adds a basic starting runbook for etcdInsufficientMembers which is a critical alert. This will be improved later on, either by etcd team or SRE folks or by feedback from our end users.

@RiRa12621 @hexfusion please take a look, as mentioned we have a lot of runbooks to add so these are just basic things and we will improve in follow ups.

### CODE REVIEW

File: alerts/etcdInsufficientMembers.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## Impact
+
+When etcd does not have a majority of instances available the Kubernetes and
+OpenShift APIs will reject read and write requests and operations that preserve
+the health of workloads cannot be performed.
+
+## Diagnosis
+
+This can occur multiple control plane nodes are powered off or are unable to
+connect each other via the network. Check that all control plane nodes are
```
Comment (hexfusion) [inline alerts/etcdInsufficientMembers.md:17]: can be a followup but I think it's a good user hint to ref firewall explicitly in network partition/issues.

Comment (lilic) [reply alerts/etcdInsufficientMembers.md:17]: Can do here, what kind of wording you were thinking? Do we have any references for this in our docs?

File: alerts/etcdInsufficientMembers.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+### Disaster and recovery
+
+If an upgrade is in progress, the alert may automatically resolve in some time
+when the master node comes up again. If MCO is not working on the master node,
+check the cloud provider to verify if the master node instances are running or not.
+
+In the case when you are running on AWS, the AWS instance retirement might need
+a manual reboot of the master node.
+
+For etcd specific issues follow the steps described in the [disaster and
```
Comment (hexfusion) [inline alerts/etcdInsufficientMembers.md:78]: We want to ensure the user understands DR is last resort vs general resolution.

File: alerts/etcdInsufficientMembers.md (hunk 1 above)
Comment (hexfusion) [reply alerts/etcdInsufficientMembers.md:17]: Not sure to be honest just in practice a lot of the "hostNetwork is broken" bugs/escalations have been firewall-related. We can work on the wording as a follow-up or even a docs bug if appropriate.

File: alerts/etcdInsufficientMembers.md (hunk 2 above)
Comment (hexfusion) [reply alerts/etcdInsufficientMembers.md:78]: could be a follow-up, just don't want a surge in DR cases from a transient condition.

Comment (lilic) [reply alerts/etcdInsufficientMembers.md]: Done, added a warning, good one!

**Review by hexfusion — COMMENTED** [review]
LGTM lets start here

**Comment by RiRa12621** [comment]
/lgtm
Thanks!

**Pull request merged**